release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -74,6 +74,15 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-networking-supported-hardware-for-sr-iov"]
+==== Supported hardware for SR-IOV
+
+{product-title} 4.12 adds support for the following SR-IOV devices:
```

Comment:
Suggestion: I think SR-IOV need an expansion to _single root I/O virtualization_ for the first usage.

Reply:
@Amrita42, great idea, thanks!